_posts/2022-12-08-developer-survey-2022.md

```diff
@@ -0,0 +1,123 @@
+
+In the frameworks compartment, for backend we are using .NET Core and ASP.NET with the .NET runtime, Spring Boot and Spring Framework with JVM, Express/node.js and Flask/Python.
+
+For frontend development, the framework ruling the set at Solita is react with almost half of the votes (45%). The runner-ups are vue.js (14%) and Angular (13%). From the clojure context, Reagent and re-frame are also getting around 10% of votes each.
+
+The variety of stacks used in projects is visible also in the most used libraries, frameworks and tools below where you can spot Java/Clojure/Python flavors, and also hints from the integrations and data engineering projects.
+
+[![Libraries](/img/developer-survey-2022/books.png)](/img/developer-survey-2022/books.png)
+
+In the Low-Code section, Solita does a number of projects in both Microsoft PowerApps and Outsystems. In addition to those, there are various one-off projects with platforms like AppSheet, Dell Boomi, MuleSoft or SalesForce Lightning.
+
+Mobile development is done mostly with modern native declarative UI frameworks (SwiftUI, Jetpack Compose), and with React Native and Flutter where cross-platform stacks are needed. Some desktop cross-platform work is also done with Electron and Qt frameworks.
+
+## Platforms and Deployment targets
+
+When our cloud projects get live, we deploy them most often in Microsoft Azure and Amazon AWS (roughly 33% for each) and Google GCP (12%). Some 10% are deployed in private cloud setups and 10% in Windows Server. A number of developers are also indicating interest towards deploying in IBM Cloud, which is currently not used in the projects.
+
+For deployment setup and configuration, we use Docker (65%), Kubernetes (15%), VirtualBox (8%), Podman (5%) and Vagrant (5%).
+
+During past two years, Solita developers have done desktop applications for Windows, Linux and MacOS, even though the volumes are much lower than in more common browser based solutions. 
+
+In the mobile context, we have done iOS/Android apps, some wearable OS and assistant solutions, plus a handful of Raspberry Pi / Arduino projects. There is also a good amount of interest among the personnel to get more work in the mobile/embedded projects.
+
+## DevOps
+
+For version control, git has swept the floor - in the survey we did not have a single vote for any other version control system. In the VCS service selection, there was more variation. GitHub was the most used (Solita is a GH corporate customer). Where the customer has other preferences, we have used Azure DevOps Services, GitLab and BitBucket.
```

Comment:
Few proposals

- git => Git
- GH corporate customer => GitHub corporate customer (for clarity)

Reply:
Fixed